roles/bwc/tasks/bwc_repos_setup.yml

```diff
@@ -63,4 +63,4 @@
     - ewc
     - enterprise
   register: bwc_repo_added
-  when: bwc_read_token != ''
+  when: bwc_read_token
```

Comment:
I guess it's different comparing to checking for empty string.
If linter is not happy with that, maybe `bwc_read_token | length > 0` would work?